### ISSUE TITLE
aria-label can be used with any typical HTML element

### ISSUE THREAD

**Issue by BDemetrix** [validator/validator#1202]
![image](https://user-images.githubusercontent.com/64604742/127847812-63dccf72-1466-4e5e-b8a9-d78d8bda0a22.png)
According to the MDN, this is not a mistake

**Comment by scottaohara** [comment]
[MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute#support) actually lists the following as elements `aria-label` should only be used on, none of which are a `div`:

- Interactive elements
- Landmark elements
- Elements that have an explicit widget role
- iframe and img elements


From your markup it looks like you're missing a `role=button` on each of those previous and next `div` elements.  General assumption would then be that they also need to be able to receive keyboard focus and thus function with the Space and Enter keys to ensure they are keyboard accessible.

**Comment by sideshowbarker** [comment]
Closing per https://github.com/validator/validator/issues/1202#issuecomment-890938817
